build.ps1

```diff
@@ -504,6 +504,7 @@ function Build-Runtime($Arch)
     -BuildDefaultTarget `
     -Defines @{
       CMAKE_INSTALL_BINDIR = Get-RuntimeInstallDir $Arch "usr\bin";
+      CMAKE_Swift_COMPILER_TARGET = $Arch.LLVMTarget;
```

Comment:
Hmm, I don't think that this matters as this is not currently honored by this build.  This _should_ generate a warning.

Reply:
Checking locally but the CI build that does the same did not generate a warning:

https://github.com/thebrowsercompany/swift-build/actions/runs/4419433944/jobs/7751702142

Reply:
It did not generate a warning locally